std/format/read.d

```diff
@@ -415,133 +526,40 @@ uint formattedRead(R, Char, S...)(auto ref R r, const(Char)[] fmt, auto ref S ar
 }
 
 /**
- * Reads a value from the given _input range according to spec
- * and returns it as type `T`.
- *
- * Params:
- *     T = the type to return
- *     input = the _input range to read from
- *     spec = the `FormatSpec` to use when reading from `input`
- * Returns:
- *     A value from `input` of type `T`
- * Throws:
- *     A `FormatException` if `spec` cannot read a type `T`
- * See_Also:
- *     $(REF parse, std, conv) and $(REF to, std, conv)
- */
-T unformatValue(T, Range, Char)(ref Range input, scope const ref FormatSpec!Char spec)
-{
-    return unformatValueImpl!T(input, spec);
-}
-
-/// Booleans
-@safe pure unittest
-{
-    import std.format.spec : singleSpec;
-
-    auto str = "false";
-    auto spec = singleSpec("%s");
-    assert(unformatValue!bool(str, spec) == false);
-
-    str = "1";
-    spec = singleSpec("%d");
-    assert(unformatValue!bool(str, spec));
-}
-
-/// Null values
-@safe pure unittest
-{
-    import std.format.spec : singleSpec;
+Reads a value from an _input range and converts it according to a
```

Comment:
```suggestion
Reads a value from the given _input range and converts it according to a
```

Reply:
Changed that in the overview table in `package.d` too.